src/components/UpcomingEventCard/index.js

```diff
@@ -27,7 +27,7 @@ const UpcomingEvents = ({ data }) => {
                 <SwiperSlide key={item.id}>
                   <div className="blog-slider_item swiper-slide">
                     <div className="blog-slider_img">
-                      <Image {...item.frontmatter.thumbnail} alt={item.frontmatter.title} />
+                      <Image {...item.frontmatter.thumbnail} imgStyle ={{maxHeight: "80rem", objectFit: "cover"}} alt={item.frontmatter.title} />
```

Comment:
Move these changes to the respective CSS file.

Reply:
Ok let me correct it